comment_scan.rb

```diff
@@ -85,16 +85,34 @@ def restart(num_to_post, bundle)
             end
           when 'huh?'
             matched_regexes = report_raw(comment["post_type"], comment["body_markdown"])
-            if matched_regexes.empty?
-              cb.say "Comment didn't match any regexes", room_id
+            reason_text = ""
+            #Go through regexes we matched to build reason_text
+            matched_regexes.each { |regex_matched| reason_text += "Matched reason \"#{Reason.where(id: regex_matched["reason_id"])[0]["name"]}\" for regex: #{regex_matched["regex"]}\n" }
+            #If post isn't deleted, check if this was an inactive comment
+            if !isPostDeleted(cli, comment["post_id"])
+              post = Array(cli.posts(comment["post_id"].to_i))[0]
```

Comment:
Pretty sure that `Array` is superfluous here. Also, might be nicer to use `first` instead of `[0]` although I don't really care either way.